checkout/react/component/MakeAccordion/index.tsx

```diff
@@ -0,0 +1,102 @@
+
+const MakeAccordion = () => {
```

Comment:
Can we call this Accordion? Just to avoid using a verb for components.

Reply:
Done